pkg/controller/humiocluster/humiocluster_controller.go

```diff
@@ -1350,7 +1434,12 @@ func (r *ReconcileHumioCluster) ensurePodsBootstrapped(ctx context.Context, hc *
 
 	r.logger.Debugf("pod ready count is %d, while desired node count is %d", podsReadyCount, hc.Spec.NodeCount)
 	if podsReadyCount < hc.Spec.NodeCount {
-		err = r.createPod(ctx, hc, foundPodList)
+		attachments, err := r.newPodAttachments(ctx, hc, foundPodList)
+		if err != nil {
+			r.logger.Errorf("failed to get pod attachments: %s", err)
+			return reconcile.Result{}, err
```

Comment:
@jswoods Given that the other `ensurePodsExist` have `return reconcile.Result{Requeue: true, RequeueAfter: time.Second * 5}, err`, perhaps we want to update this one to do the same?